### PR TITLE
URLLoader issue related with empty data for POST request

### PR DESCRIPTION
**Pull request by nihavend**
While using URLLoader load method, the data in request was not being set during POST request. Fixed. @aharui for your review

### CODE REVIEW

File: frameworks/projects/MXRoyale/src/main/royale/mx/rpc/http/AbstractOperation.as [hunk 1]
```diff
@@ -856,8 +856,9 @@ package mx.rpc.http
 			}
 			
 			if (ctype == CONTENT_TYPE_XML)
-			{/*
-				if (parameters is String && xmlEncode == null)
+			{
+				paramsToSend = parameters.toXMLString();
```
[the remark this answers is not in the document]
Comment (nihavend) [reply frameworks/projects/MXRoyale/src/main/royale/mx/rpc/http/AbstractOperation.as]: I did not test the case on minified version at the time I created PR as I am worhing on debug version. 
I just tried to test it but, I had prior errors before coming to this line ("window.base64js.om is not a function") and can not go forwad.

One option, to leave what I am doing and try to find out all of the prior errors on minified version and try to to answer your question.

[the remark this answers is not in the document]
Comment (nihavend) [reply frameworks/projects/MXRoyale/src/main/royale/mx/rpc/http/AbstractOperation.as]: @Harbs do youl like me to add these changes to this PR?

[the remark this answers is not in the document]
Comment (Harbs) [reply frameworks/projects/MXRoyale/src/main/royale/mx/rpc/http/AbstractOperation.as]: I made those change already.
Thanks.

[the remark this answers is not in the document]
Comment (nihavend) [reply frameworks/projects/MXRoyale/src/main/royale/mx/rpc/http/AbstractOperation.as]: Okay @Harbs thakn you. Do yu want me to change the AbstractOperation class ? `(parameters as XML).toXMLString();`

[the remark this answers is not in the document]
Comment (nihavend) [reply frameworks/projects/MXRoyale/src/main/royale/mx/rpc/http/AbstractOperation.as]: I checked the code after your modifications on Base64 wrapper and can confirm that `paramsToSend = parameters.toXMLString();` works as expected without explicit casting. What do you suggest ?

[the remark this answers is not in the document]
Comment (nihavend) [reply frameworks/projects/MXRoyale/src/main/royale/mx/rpc/http/AbstractOperation.as]: @Harbs just for reminder.

[the remark this answers is not in the document]
Comment (Harbs) [reply frameworks/projects/MXRoyale/src/main/royale/mx/rpc/http/AbstractOperation.as]: Thanks for the reminder and thanks for checking!

**Pull request merged**